README.md

```diff
@@ -29,6 +26,7 @@ Additional features includes [Annotations](https://opengeoscience.github.io/geoj
 [Animated Transitions](https://opengeoscience.github.io/geojs/examples/transitions), and
 [Widgets](https://opengeoscience.github.io/geojs/examples/widgets).
 
+GeoJS can also be used for visualizing medical imagery. See [HistomicsTK](https://github.com/DigitalSlideArchive/HistomicsTK) for an example.
```

Comment:
We should probably point this to [Digital Slide Archive](https://digitalslidearchive.github.io/digital_slide_archive)